content/en/docs/concepts/storage/volumes.md

```diff
@@ -1235,6 +1235,40 @@ Kubernetes component using the following feature gate flags:
 Learn how to
 [setup your PV/PVC with raw block volume support](/docs/concepts/storage/persistent-volumes/#raw-block-volume-support).
 
+#### CSI inline volume support
+
+{{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+
+This feature is to enable support for CSI inline ephemeral volumes to be embedded
+within a pod spec as illustrated in the following YMAL:
+
+```yaml
+kind: Pod
+apiVersion: v1
+metadata:
+  name: my-csi-app
+spec:
+  containers:
+    - name: my-frontend
+      image: busybox
+      volumeMounts:
+      - mountPath: "/data"
+        name: my-csi-inline-vol
+      command: [ "sleep", "1000000" ]
+  volumes:
+    - name: my-csi-inline-vol
+      csi:
+        driver: inline.storage.kubernetes.io
+        volumeAttributes:
+              foo: bar
+```
+
+```
+--feature-gates=CSIInlineVolume=true
+```
+
+Read about CSI inline support [here](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/20190122-csi-inline-volumes.md).
```

Comment:
I don't know how relevant the design document is to users?

Reply:
Removed